lib/adapter.js

```diff
@@ -48,12 +48,12 @@ class KnexAdapter {
   createPolicy(ptype, rule) {
     return {
       ptype,
-      v0: rule[0],
-      v1: rule[1],
-      v2: rule[2],
-      v3: rule[3],
-      v4: rule[4],
-      v5: rule[5]
+      v0: typeof rule[0] !== 'undefined' ? rule[0] : null,
```

Comment:
Let's add a helper function for that instead of doing the same over and over

```
const valueOrNull = value => typeof value !== 'undefined' ? value : null
...
v0: valueOrNull(rule[0])
```